### ISSUE TITLE
Clearify release process and clean up stale branches

### ISSUE THREAD

**Issue by mvhirsch** [thumbor/thumbor#1388]
Hi,

I really appreciate your efforts and really like thumbor :heart:

That said, please clean up stale branches ([check here](https://github.com/thumbor/thumbor/branches)).
Also it seems you do not delete branches "on merge" by default. Maybe [this helps (it's just a setting in github)](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches).

It's really hard and confusing to find anything where to contribute to. This leads to questions like https://github.com/thumbor/thumbor/issues/1371. The [contribution guide](https://github.com/thumbor/thumbor/blob/master/CONTRIBUTING.md) won't help on it either, atm.

Sadly this leads to my situation, where I do not know which branch is the actual "master". Even releases of previous versions are confusing me: 

* v7.0.0 was released from `master`
* [6.7.5 was released](https://github.com/thumbor/thumbor/releases/tag/6.7.5) from "fix/6.7" 
* whereas [`6.6.1` was released](https://github.com/thumbor/thumbor/releases/tag/6.6.1) on `master`

Not being able to reproduce a build/package is always problematic for endusers (and reduces trust on package managers).

May I ask what's the current branching- and release-pattern?

**Comment by heynemann** [comment]
Stale branches (mostly) cleared. Thanks for the reminder!

**Comment by heynemann** [comment]
First I want to apologize for the confusion. It should be clearer to anyone contributing with thumbor how to do it. I'll update the contributing document to reflect that.

Second, to answer your question, 6.6.1 was released before master branch was python 3 only, thus makes sense to use master. After we upgraded thumbor master to python 3 we could not release python 2 from it anymore, thus the purpose of the fix/6.7 branch. Whenever we find anything that must be backported we'll update that branch.

So to summarize:
* New features go in the master branch and apply ONLY to the 7.0.0+ versions (as we're not adding new features to 6.x anymore);
* Bug fixes in the python3 version go in the master branch;
* Bug fixes in the python2 version go in the fixes/6.7.x branch.

Does this clarify things for you? I'll add it to the contributing document if it does (or you could contribute it :D).

**Comment by heynemann** [comment]
Just added stale bot to thumbor to make sure we're not in this same state again :) Thanks for the help in cleaning things up!

**Comment by mvhirsch** [comment]
Awesome, thank you very much!

Yes, I'll add some :heart: to the contribution docs.
